### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by pedja76** [cedricp/ddt4all#1242] [Q&A]
I ask for another help. GRAND SCENIC III (2016) is Ph2 or not? (x95 or x95ph2)?

**Comment by pedja76** [comment]
My respect to all.
I would like someone to help me.
I want to use the ddt4all app for my car but I don't know which car model to choose. I own a GRAND SCENIC III (2016).
Is this a Ph2 phase vehicle or not (X95 or X95Ph2)?

**Reply by AndyIreland** [reply]
dont choose anything, just leave the vehicle as 'choose vehicle' and then press the magnifying glass in ddt4all and choose 'CAN' then let ddt4all find your ECU's on your car
